js/hview.js

```diff
@@ -557,20 +584,17 @@ $(function () {
         }
     }
 
-    const renderList = function (root, vListContent = $("#vlist_content").empty(), boxContent = $("#border-box").empty()) {
+    const renderList = function (root, vListContent = document.findElementById("#vlist_content"), boxContent = document.findElementById("#border-box")) {
```

Comment:
either use
   document.queryselector("#vlist_content")
of use
  document.findElementById("vlist_content") // without the leading `#`

Reply:
Done. (Btw, I tested this, and it is actually getElementById("something"). It has been fixed and updated.